packages/documentation/src/stories/accessibility-practices/foundational-structure-and-semantics/reference-crossing-the-shadowdom/aria-describedby/examples/workingExample-IIa.sample.html

```diff
@@ -0,0 +1,16 @@
+<span id="descriptionId">My Description</span>
```

Comment:
In the first two examples on the page you implement the description after the element that references it.
Should we keep the order to make it more comparable or is it mandatory to have the referenced elmeent before the one which references it?